pkg/oam/util/test_utils.go

```diff
@@ -102,6 +103,7 @@ func (matcher ErrorMatcher) FailureMessage(actual interface{}) (message string)
 }
 
 // NegatedFailureMessage builds an error message.
+//nolint:errorlint
```

Comment:
why?

Reply:
Linter warns that it should use errors.As() instead of type assertion for it will fail on wrapped error. But if I use `errors.As()` to refactor the func, it may result in unpredictable issues since it's a util func. So I just comment nolint for now. 
BTW, the same issues happen in the project several times, I would like to fix these `nolint:errorlint` latter.

Reply:
I see, thanks for your explain, please add TODO and explain in it,thanks!